graphics/Pangolin/Portfile

```diff
@@ -19,9 +19,20 @@ long_description    Pangolin is a lightweight portable rapid development library
                     Pangolin also provides a mechanism for manipulating program variables through config files \
                     and ui integration, and has a flexible real-time plotter for visualising graphical data.
 
-depends_lib-append  port:glew
+depends_build-append \
+                    port:pkgconfig
+
+depends_lib-append  port:glew \
+                    port:openexr \
+                    port:libdc1394 \
+                    path:lib/libavcodec.dylib:ffmpeg \
+                    path:lib/libavformat.dylib:ffmpeg \
+                    path:lib/libavutil.dylib:ffmpeg \
+                    path:lib/libswscale.dylib:ffmpeg
```

Comment:
A dependency on a port should only be added once. Most ports that depend on ffmpeg do so using `path:lib/libavcodec.dylib:ffmpeg` so that's the one I'd use here.

Reply:
Done